spec/graphql/queries/requests/encounter_details_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+module Queries
+  module Encounters
+    RSpec.describe Encounter, type: :request do
+      describe 'encounter' do
+        it 'returns details for a single encounter' do
+          encounter = FactoryBot.create(:encounter)
+          FactoryBot.create_list(:encounter_monster, 3, encounter: encounter)
+    
+          post '/graphql', params: { query: query, variables: { id: encounter.id } }
```

Comment:
maybe this is a way we test the create endpoint, using variables, rather than hard coded query.